post/models.py

```diff
@@ -24,7 +24,7 @@ def __str__(self):
 
 class Comment(models.Model):
     post = models.ForeignKey(Post, related_name="comments", on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    #user = models.ForeignKey(User, on_delete=models.CASCADE)
     body = models.TextField()
```

Comment:
commented code that won't be used can be deleted

Reply:
Why are you deleting user from model? We need to track who wrote the comment.

Reply:
later on we will make only user who log in  can see posts , right ? which means whoever can see post , can comment, i wrote something in the code , saying if the user is authenticated , there is a java script will show the name from user_id ... if I didn't delete the user model , it doesn't work

Reply:
If we don't save user in DB table for comment there is no way to retrieve this information later with JavaScript.